services/pane.js

```diff
@@ -91,36 +90,40 @@ function createPublishMessages(res) {
  * @returns {Element}
  */
 function createPublishActions(res) {
-  var today = moment().format('YYYY-MM-DD'),
+  var actions = dom.find('.publish-actions'),
```

Comment:
#1 (update) looking at support for the client side and `const` is still a bit problematic. Safari likes in non-strict mode, chrome likes w/ strict. Moving forward with `var` for now.